setup.py

```diff
@@ -1,12 +1,12 @@
-from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'rqt_console'
 
 setup(
     name=package_name,
     version='1.1.1',
-    packages=find_packages(),
+    packages=[package_name, package_name + '.filters'],
```

Comment:
IIUC you can continue using `find_packages()` here (but I would test locally if `rqt_reconfigure` is happy with that).
The issue was that the package is nested in the `src` directory, in other places our folder structure is `rqt_console/rqt_console/__init__.py` instead of `rqt_console/src/rqt_console/__init__.py`.

In this case, with the `src` folder, you need the `package_dir` option below.
Sorry for the error in the previous review.

There's a `package_dir` option in `setup.cfg` if you prefer that one: https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html?highlight=find_packages#using-find-or-find-packages.